tests/v2/testlib/common.nim

```diff
@@ -0,0 +1,38 @@
+import
+  std/times
+import
+  ../../../waku/v2/protocol/waku_message,
+  ../../../waku/v2/utils/time
+
+const 
+  DefaultPubsubTopic* = "/waku/2/default-waku/proto"
+  DefaultContentTopic* = ContentTopic("/waku/2/default-content/proto")
+
+
+proc now*(): Timestamp =
```

Comment:
Looks great! cant believe the amount of duplicated `now()` and `fakeWakuMessage()` definitions.

As an idea, perhaps we want `now()` to be defined in `waku/v2/utils/time.nim`? We have also `nowInUnixFloat()` there.

Reply:
Accidentally saw more, in case you want to move it to `time.nim` and use it in prod code.

```console
$ git grep "getNanosecondTime(getTime().toUnixFloat())"
waku/v2/node/storage/message/message_retention_policy_time.nim:  let now = getNanosecondTime(getTime().toUnixFloat())
waku/v2/node/storage/message/queue_store/queue_store.nim:    now = getNanosecondTime(getTime().toUnixFloat())
waku/v2/protocol/waku_store/message_store.nim:                      else: getNanosecondTime(getTime().toUnixFloat())
waku/v2/protocol/waku_store/protocol.nim:    now = getNanosecondTime(getTime().toUnixFloat())
waku/v2/protocol/waku_store/protocol.nim:                        else: getNanosecondTime(getTime().toUnixFloat())
waku/v2/protocol/waku_store/protocol.nim:    now = getNanosecondTime(getTime().toUnixFloat())
```

Reply:
> > As an idea, perhaps we want now() to be defined in waku/v2/utils/time.nim? We have also nowInUnixFloat() there.

This is out of the scope of this PR. This PR focuses mainly on extracting the common methods of the tests.

There is still work to be done regarding the `Timestamp`. For example, we should stop using `std/times` and use the `chronos/timer` module instead; the `utils` module is not the right place for timestamp types (we need some kind of "core" module to hold that); etc.

Reply:
sure!